src/sections/Meshmap/meshmap.style.js

```diff
@@ -2,7 +2,12 @@ import styled from "styled-components";
 
 const MeshmapWrapper = styled.div`
   
-
+  .supscript{
+    --theme-color:#00d3a9;
+    color: var(--theme-color);
```

Comment:
No, this isn't what I mean. I mean use the central theme - https://github.com/layer5io/layer5/blob/master/src/theme/app/themeStyles.js

Reply:
Like `color: ${props => props.theme.black};` here the variable `black` is defined in the themeStyles.js file, which Lee mentioned abovee.

Reply:
But for this case, you need to use `color: ${props => props.theme.caribbeanGreenColor};`

Reply:
ok got it.